src/HybridDb.Tests/DocumentMigratorTests.cs

```diff
@@ -86,6 +85,19 @@ public void Bug_DontCallTypeMapperDirectly()
             Should.NotThrow(() => new DocumentMigrator(configuration).DeserializeAndMigrate(null, design, Row(NewId(), document, 0, design.Discriminator)));
         }
 
+        [Fact]
+        public void ReturnsNullIfDocumentIsNull()

```

Comment:
I dont understand this test name. Its test that we do not throw if document is null?

Reply:
redundant test, should be removed.